crates/merkle/ssz/mmr.ssz

```diff
@@ -0,0 +1,11 @@
+# SSZ schema definitions for MMR types
+# These types are used for SSZ serialization of Merkle Mountain Range structures
+
+### Compact representation of the MMR that can hold upto 2**64 elements.
+class CompactMmr64(Container):
+    ### Total number of entries inserted into the MMR.
+    entries: uint64
+    ### Logarithm base 2 of the capacity (number of peaks).
+    cap_log2: uint8
+    ### Peak roots of the MMR.
+    roots: List[Vector[byte, 32], 1024]
```

Comment:
Use `Bytes32` and 64 since we can't count higher than the maximum number of entries.